spec/components/onboarding_signal_splash_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe OnboardingSignalSplash::OnboardingSignalSplash, type: :component do
+  subject { render_inline(described_class.new(**params)) }
+
+  let(:params) { {} }
+  it { should have_css('.OnboardingSignalSplash') }
```

Comment:
We might want to think about a better name than splash screen.

Reply:
+1, I found the naming confusing when I first saw it. But I also don't have better ideas.
As the page serves to get Contributors to send a first message via Signal something like `OnboardingSignalMessagePage`, `OnboardingSignalMessageScreen` or `OnboardingSignalMessageView` might be fitting, but don't know if that conveys the meaning better.

Reply:
I think I’m going to rename this to `OnboardingSignalLink` -- this is what we call the respective component in the Telegram flow as well.